### PR TITLE
Pricing Grid: update copy for multi-site management

### PR DESCRIPTION
**Pull request by aneeshd16**
Fixes Automattic/martech#1620

## Proposed Changes

* Changes the copy from "Multi-site management" to "Centralized Site Management".

## Testing Instructions

* Go to `/start/plans`.
* On the plans step, click on the "Compare plans" button.
* Scroll to the developer tools section and confirm that the copy is correct.
image

Once the PR is reviewed, I'll add the String Freeze label to this PR.

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: client/lib/plans/features-list.tsx
Comment (claudiucelfilip) [inline client/lib/plans/features-list.tsx:1838]: The new string doesn't look like it's translated yet.
We should wrap either this in a translation check or `String Freeze` in this PR.

Comment (aneeshd16) [reply client/lib/plans/features-list.tsx:1838]: Yes, I'll add the string freeze label once this is approved. (Had mentioned that in the PR description 🙂 )

Comment (aneeshd16) [reply client/lib/plans/features-list.tsx:1838]: Probably not much to review here, I'll go ahead and add the label. 😆

**Review by claudiucelfilip — APPROVED** [review]
LGTM once it gets translated.

**Pull request merged**
